au4/src/trackedit/internal/au3/au3interaction.cpp

```diff
@@ -733,10 +733,14 @@ bool Au3Interaction::trimClipLeft(const ClipKey& clipKey, secs_t deltaSec)
     trackedit::ITrackeditProjectPtr prj = globalContext()->currentTrackeditProject();
     prj->onClipChanged(DomConverter::clip(waveTrack, clip.get()));
 
+    if (completed) {
+        projectHistory()->pushHistoryState("Clip trimmed", "Trim clip");
```

Comment:
In AU3, the history item shows the amount by which the clip was trimmed (you can look for `"Trim by %.02fs"`)
When there are behavioral changes, I'd always refer to @LWinterberg or @Tantacrul to find out if it's ok.

Reply:
Answered by Leo in person: no.